support-workers/src/main/scala/com/gu/emailservices/SupporterPlusEmailFields.scala

```diff
@@ -30,6 +41,7 @@ class SupporterPlusEmailFields(
         "billing_period" -> state.product.billingPeriod.toString.toLowerCase,
         "product" -> s"${state.product.billingPeriod.toString.toLowerCase}-supporter-plus",
         "zuorasubscriberid" -> state.subscriptionNumber,
+        "subscription_details" -> subscription_details,
```

Comment:
currently I've called it subscription_details but I noticed it was different between paper and digisub, so it could be subscription_rate instead if we prefer consistency that way